app/scripts/controllers/mmi-controller.ts

```diff
@@ -70,8 +69,6 @@ export default class MMIController extends EventEmitter {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public keyringController: any;
 
-  public preferencesController: PreferencesController;
```

Comment:
Is this change related to `AppStateController`?
I think that this PR joins the `AppStateController` typescript conversion, an initial migration to BaseControllerV2, and also some post-merge work derived from PreferencesController

If that's correct, perhaps splitting these changes in multiple specific PR would make it easier to go through all the changes